### ISSUE TITLE
Losing too much information?

### ISSUE THREAD

**Issue by KLemiesz** [KLemiesz/sds192-mp2#2]
Hey guys, I'm wondering if we're losing too much information by averaging each state instead of each party within each state? I.e. we can say which state overall donated the most, but we can't determine which party that may have been for.

**Comment by emmautz** [comment]
Hey Katie, that makes sense. Maybe we could further break down the top 3 or 5 states into parties? The thing about the party affiliations in the data is that there are very few committees that have a party affiliation, so when I wrote that code that looked at donations by state AND party, it was evident that most states only had a couple parties reported (sometimes none).

**Comment by KLemiesz** [comment]
Yeah, I think maybe choosing some states to compare then could work, I just think we've filtered it to a point where we may not actually be able to analyse like we want to

**Comment by KLemiesz** [comment]
Okay, I switched up the bottom code that I think you wrote Emma(?) to be a function and applied it to `newTable`, which doesn't have the na's thanks to JiYoung. I created state_lst to add the states whose party contributions we want to look at, which we could pick by first comparing the full state contributions in the graph and seeing which we want more detail on.

**Comment by emmautz** [comment]
Sounds great Katie! I say we present this all to our peers tomorrow in class and ask for their feedback about it as well. I can flesh out our explanation with some additional info I hope, and then we can pull everything together this weekend?
